### PR TITLE
Fixed compilation of test code

### PR DESCRIPTION
**Pull request by paullepoulpe**

### CODE REVIEW

File: src/test/scala/devsearch/FeatureRetrieverTest.scala [hunk 1]
```diff
@@ -14,7 +14,7 @@ class FeatureRetrieverTest extends FlatSpec with Matchers {
       val l = FeatureRetriever.get(Seq())
       val l2 = FeatureRetriever.get(List("variableDeclaration=rest type=devsearch.ast.PrimitiveTypes.Int$"))
 
-      assert(l.isEmpty())
+      assert(l.count() == 0)
```
Comment (zommerfelds) [inline src/test/scala/devsearch/FeatureRetrieverTest.scala:17]: What is the difference?

Comment (paullepoulpe) [inline src/test/scala/devsearch/FeatureRetrieverTest.scala:17]: The first one doesn't compile

Comment (zommerfelds) [inline src/test/scala/devsearch/FeatureRetrieverTest.scala:17]: `isEmpty` was apparently introduced on Spark 1.3.0.

**Comment by zommerfelds** [comment]
Nice branch name ;-)

**Pull request merged**